python/hail/docs/tutorials/01-genome-wide-association-study.ipynb

```diff
@@ -435,11 +433,7 @@
    "outputs": [],
    "source": [
     "dp_hist = ds.aggregate_entries(agg.hist(ds.DP, 0, 30, 30))\n",
-    "p = figure(title='DP Histogram', background_fill_color='#EEEEEE')\n",
-    "p.quad(bottom=0, top=dp_hist.bin_freq, \n",
-    "       left=dp_hist.bin_edges[:-1], right=dp_hist.bin_edges[1:],\n",
-    "       legend = 'DP', line_color='black')\n",
-    "show(p)\n"
+    "plot.histogram(dp_hist, title='DP Histogram', label='DP')"
```

Comment:
So just to reiterate our conversion, @tpoterba you're suggesting something like:

```
p = hl.plot.histogram(dp_hist)
p.title.text = 'DP Histogram'
p.yaxis.axis_label = 'DP'
show(p)
```

?  Maybe there is a more natural way to write this in Bokeh, but I have to say, I prefer the title/label arguments, in which case it would become:

```
# maybe legend='DP' is better to match quad interface?
# although I don't see legend documented as an option for quad()
p = hl.plot.histogram(dp_hist, title='DP Histogram', label='DP')
show(p)
```

Reply:
In the call rate and mean DP scatterplot from the tutorial, the old code extended the renderers to indicate outlier cuts. I originally had this in my method as a boolean parameter, but I think this is an example of where the user can extend the plot after doing the function (i.e. manually adding the Span renderers to show outlier plots). Which do you guys prefer in this case?

Reply:
I'm OK with title and axes label arguments in the plot function

Reply:
and in this case, the tutorial code should manually add the Spans. that's a pretty uncommon thing to do.